indra/sources/signor.py

```diff
@@ -194,24 +216,61 @@ def __init__(self, signor_csv=None, delimiter='\t'):
 
     @staticmethod
     def _get_agent(ent_name, ent_type, id, database):
```

Comment:
This function doesn't need to return a list of Agents for complexes--rather, it should return a single agent with the other agents incorporated as BoundConditions. For simplicity, it's fine to just use the first protein listed in the complex as the "main" agent, the rest as BoundConditions.